proof/refine/ARM/Finalise_R.thy

```diff
@@ -3032,6 +3017,118 @@ crunches schedContextCompleteYieldTo, unbindNotification, unbindMaybeNotificatio
   for sch_act_simple[wp]: sch_act_simple
   (simp: crunch_simps wp: crunch_wps rule: sch_act_simple_lift)
 
+lemma schedContextZeroRefillMax_unlive[wp]:
```

Comment:
This isn't the weakest precondition, is it? Can't we just have that there's an sc at scPtr, and `schedContextZeroRefillMax` will make it unlive?

Reply:
scRefillMax doesn’t change the liveness, I think. So, this should be ok.

Reply:
Yeah, what Miki said. `finaliseCap` actually makes scheduling contexts unlive over a couple of steps, which is what `schedContextUnbindYieldFrom_makes_unlive` and the following `obj_at'` lemmas are showing.